src/HubBootstrapper.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Sentry;
+
+use Yiisoft\Yii\Sentry\Http\YiiRequestFetcher;
+use Psr\Container\ContainerInterface;
+use Psr\Log\LoggerInterface;
+use RuntimeException;
+use Sentry\ClientBuilder;
+use Sentry\Integration as SdkIntegration;
+use Sentry\Integration\IntegrationInterface;
+use Sentry\Options;
+use Sentry\SentrySdk;
+use Sentry\State\HubInterface;
+use Sentry\Transport\TransportFactoryInterface;
+
+use function is_string;
+
+final class HubBootstrapper
+{
+    public const DEFAULT_INTEGRATIONS = [];
+
+    public function __construct(
+        private Options $options,
+        private YiiSentryConfig $configuration,
+        private TransportFactoryInterface $transportFactory,
+        private LoggerInterface $logger,
+        private HubInterface $hub,
+        private ContainerInterface $container,
+    ) {
+    }
+
+    public function bootstrap(): void
+    {
+        $this->options->setIntegrations(fn (array $integrations) => $this->prepareIntegrations($integrations));
+
+        $clientBuilder = new ClientBuilder($this->options);
+        $clientBuilder
+            ->setTransportFactory($this->transportFactory)
+            ->setLogger($this->logger);
+
+        $client = $clientBuilder->getClient();
+
+        $hub = $this->hub;
+        $hub->bindClient($client);
+        SentrySdk::setCurrentHub($hub);
+    }
+
+    /**
+     * @param IntegrationInterface[] $integrations
+     *
+     * @return IntegrationInterface[]
+     */
+    public function prepareIntegrations(array $integrations)
+    {
+        $userIntegrations = $this->resolveIntegrationsFromUserConfig();
+        if ($this->options->hasDefaultIntegrations()) {
+            $integrations = array_filter(
+                $integrations,
+                static function (
+                    SdkIntegration\IntegrationInterface $integration
```

Comment:
Maybe it's not documented and not the part of the linter, but based on code style I've seen, we don't add new lines in such cases (arguments, if blocks with `instanceof`checks below, etc.). If it fits to one line (max line length of 120 characters is not exceeded) and especially if there are only 1 argument / 1 condition - then new lines are not needed. arrays probably is the only exception.